client/my-sites/plans-v2/constants.ts

```diff
@@ -164,3 +165,28 @@ export const SELECTOR_PLANS = [
 	PLAN_JETPACK_COMPLETE,
 	PLAN_JETPACK_COMPLETE_MONTHLY,
 ];
+
+export const DAILY_PLAN_TO_REALTIME_PLAN: Record<
+	JetpackOfferDailyPlans,
+	JetpackOfferRealtimePlans
+> = {
+	[ PLAN_JETPACK_SECURITY_DAILY ]: PLAN_JETPACK_SECURITY_REALTIME,
+	[ PLAN_JETPACK_SECURITY_DAILY_MONTHLY ]: PLAN_JETPACK_SECURITY_REALTIME_MONTHLY,
+};
+
+/**
+ * List of plans and products that can be upgraded from daily to real-time
+ * through an upgrade nunge.
+ */
+export const UPGRADEABLE_WITH_NUDGE = [
+	PLAN_JETPACK_SECURITY_DAILY,
+	PLAN_JETPACK_SECURITY_DAILY_MONTHLY,
+	// This is only for testing purposes to make the nudge appear inside the
+	// Jetpack Security bundle card. This because at the moment we can't purchase
+	// Jetpack Security Daily.
+	// TODO: remove this once we can purchase the new plans.
+	'jetpack_security',
```

Comment:
This is already defined as `OPTIONS_JETPACK_SECURITY` above specifically as we need a pseudo product like this.